CarRentals/CarRentals/DAL/ProgramOptions.cs

```diff
@@ -8,7 +8,9 @@ namespace CarRentals
 {
     public class ProgramOptions
     {
-        public static string sectionName = "JsonFile";
+        public static string sectionCarsName = "JsonFile:CarsJsonFile";
+        public static string sectionCustomersName = "JsonFile:CustomersJsonFile";
+        public static string sectionRentalsName = "JsonFile:RentalsJsonFile";
```

Comment:
No need to do this. You should leave the previous line (SectionName) as it was and all these should become properties of a new options class. See the official docs for how to address this.